### ISSUE TITLE
Fix Search Automated Checks (Accessibility Insights for Web)

### ISSUE THREAD

**Issue by joelvdavies** [ral-facilities/datagateway#923]
**Description**:
There are various fixable issues found on the search page when running automated checks using the Accessibility Insights for Web extension.

- [x] Date picker buttons need aria-labels:
![image](https://user-images.githubusercontent.com/90245114/141462111-42078bda-8e45-4167-b293-ef5b99a3fbd5.png)
- [x] Filters fail `duplicate-id-aria` in table view
- [x] 'Sort by' filters fail `duplicate-id-aria` in card view
- [x] Investigate strange floating 'duplicate-id-aria' in card view (see screenshot) - Update: This is from the hidden advanced filters
- [x] `aria-valid-attr-value` for investigation tab in card view only (see screenshot)
![image](https://user-images.githubusercontent.com/90245114/141462422-050b008a-0378-4e48-91fc-2f7620522cc7.png)
- [x] Adding filter by investigation/facility id fails `listitem` as list elements not defined in a list:
![image](https://user-images.githubusercontent.com/90245114/141462722-31c75cd4-203f-401f-a7dc-77c995a4166b.png)

**Acceptance criteria**:
- [ ] Above issues fixed/investigated
